lib/ansible/cli/galaxy.py

```diff
@@ -901,10 +896,9 @@ def execute_info(self):
             gr = GalaxyRole(self.galaxy, self.api, role)
 
             install_info = gr.install_info
-            if install_info:
-                if 'version' in install_info:
-                    install_info['installed_version'] = install_info['version']
-                    del install_info['version']
+            if install_info and 'version' in install_info:
```

Comment:
this is not equal since before if "'version' in install_info" was false it would still run "role_info.update(install_info)"